test/Microsoft.Sbom.Api.Tests/Microsoft.Sbom.Api.Tests.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.Sbom.Api\Microsoft.Sbom.Api.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.Sbom.Parsers.Spdx30SbomParser\Microsoft.Sbom.Parsers.Spdx30SbomParser.csproj" />
```

Comment:
The asymmetry of including SPDX30 but not SPDX22 makes me wonder if we're overlooking some test cases

Reply:
SPDX30 is included because I use `using Generator30 = Microsoft.Sbom.Parsers.Spdx30SbomParser.Generator;` in `\sbom-tool\test\Microsoft.Sbom.Api.Tests\Workflows\ManifestGenerationWorkflowTests.cs`. However, for 2.2 the existing code uses stubs and some hardcoded logic in `TestManifestGenerator`. 

I thought that using the actual generator (Spdx30SbomParser.Generator) would be give better code coverage which is why I made this choice.

Reply:
Not sure I follow this comment. Test coverage for the SPDXParser project should be based on code in the SPDXParserTests project, not on the ApiTests project. The ApiTests project should be testing code in the Api project and should be independent of any downstream dependencies.

Reply:
This test was meant to test the generation workflow with the Spdx30SbomParser.Generator. But since that means adding a dependency, I think it would be best to remove it here and include it as part of E2E/integration tests that I plan to write.

Reply:
Talked offline, since this is a test project it can be refactored later.

Reply:
As discussed, let's wait for a later PR to refactor the tests